assets/questions/plusone.js

```diff
@@ -27,7 +27,9 @@ globalData.questions['plusone'] = {  // eslint-disable-line dot-notation
   auxiliaryCode: {
     python:
 `class AuxiliaryCode(object):
-    pass
+    @classmethod
+    def foo(cls, array):
```

Comment:
This isn't an actual buggy test, though. Please write code to support a buggyOutputTest.